packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -34,24 +30,14 @@ export const SafeLocalStorage = {
     value: SafeLocalStorageItems[Key]
   ): void {
     if (isSafe()) {
-      localStorage.setItem(key, JSON.stringify(value))
+      localStorage.setItem(key, value)
     }
   },
   getItem<Key extends keyof SafeLocalStorageItems>(
     key: Key
   ): SafeLocalStorageItems[Key] | undefined {
     if (isSafe()) {
-      const value = localStorage.getItem(key)
-
-      if (value) {
-        try {
-          return JSON.parse(value)
-        } catch (e) {
-          console.warn('Error parsing value from localStorage', key, e)
-
-          return undefined
-        }
-      }
```

Comment:
We'd done this change to have typed values + prevent having to parse and stringify all the time 

Why remove this?